source/edge-server/monitor.txt

```diff
@@ -0,0 +1,232 @@
+- Through Admin API calls
+
+You can also forward logs to a MongoDB collection or an external service.
+
+This page contains details about how to view metrics and logs for your 
+Edge Server instances, and what information is available.
+
+Before You Begin
+----------------
+
+.. tabs::
+
+   .. tab:: Atlas UI
+      :tabid: atlas-ui
+
+      You need the following to manage Edge Servers in the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+   
+   .. tab:: Local UI
+      :tabid: local-ui
+
+      The local UI is enabled by default, and only requires a running
+      Edge Server instance.
+
+      On an untrusted host, you can disable the local UI by running the
+      following ``edgectl`` command:
+
+      .. code-block:: console
+
+         edgectl config --app-id=<YOUR-APP-ID> --disable-local-ui
+
+   .. tab:: Admin API
+      :tabid: api
+
+      You need the following to manage Edge Servers with the Admin API:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/api-project-app-id.rst
+
+.. _edge-server-status:
+
+Get Edge Server Instance Status
+-------------------------------
+
+You can get details about the status of Edge Server instance in a few ways:
+
+- View the Edge Server instance details in the Atlas UI
+- View the status details in the local UI
+- Use ``edgectl`` to check the status
+- Make a request to the relevant Admin API endpoint
+
+The available details vary depending on how you access them.
+
+.. tabs::
+
+   .. tab:: Atlas UI
+      :tabid: atlas-ui
+
+      .. include:: /includes/edge/atlas-ui-get-edge-server-list-details.rst
+
+      The information available when viewing an Edge Server instance vary
+      depending on the instance state. For details, refer to 
+      :ref:`edge-server-details`.
+   
+   .. tab:: Local UI
+      :tabid: local-ui
+
+      To access the local UI, open a web browser and go to ``localhost``
+      on the ``http-port``. The default ``http-port`` is ``80``, so
+      you can access the local UI at:
+
+      ``http://localhost:80``
+
+      You can change the port by providing an alternative port as an ``Int`` 
+      with the following ``edgectl`` flag:
+
+      .. code-block:: console
+
+         edgectl config --http-port=<YOUR-PORT-INT>
+
+   .. tab:: edgectl
+      :tabid: edgectl
+
+      .. include:: /includes/edge/edgectl-status.rst
+
+   .. tab:: Admin API
+      :tabid: api
+
+      .. include:: /includes/edge/admin-api-get-edge-server-status.rst
+
+.. _edge-server-logs:
+
+Edge Server Logs
+----------------
+
+Edge Server uses the following log types:
+
+- :ref:`authentication-logs`: When an Edge Server instance attempts to 
+  authenticate with Atlas, or clients attempt to authenticate with an Edge
+  Server instance, this generates an authentication log.
+- :ref:`logs-sync`: When an Edge Server instance syncs with Atlas, or an 
```

Comment:
starting monday, you'll also see logs when device connect to and auth with the edge server, provided that the edge server is able to send those logs up